spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -799,6 +799,9 @@ def write_data_spec(
 
         self._weight_scales[placement] = weight_scales
 
+    def clear_connection_cache(self):
+        self._retrieved_blocks = dict()
```

Comment:
This doesn't fix any issues with plasticity itself.  We need a different test for that...